torchgeo/models/dofa.py

```diff
@@ -267,8 +274,17 @@ def __init__(
         """
         super().__init__()
 
+        self.img_size = img_size
+        self.patch_size = patch_size
+        self.drop_rate = drop_rate
+        self.embed_dim = embed_dim
+        self.depth = depth
+        self.num_heads = num_heads
         self.dynamic_embed_dim = dynamic_embed_dim
+        self.num_classes = num_classes
         self.global_pool = global_pool
+        self.mlp_ratio = mlp_ratio
+
         if self.global_pool:
             norm_layer = norm_layer
             embed_dim = embed_dim
```

Comment:
Do these lines do anything? We can probably delete them (could be in a separate PR)

Reply:
It does not look like it, but did not want to touch other parts beyond saving the attributes.